### PR TITLE
Add doc about Ntfy

### PR DESCRIPTION
**Pull request by p1gp1g**
Make it first class one
Create FAQ to clear the index

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -3,37 +3,25 @@ title: UnifiedPush
...
-You should visit the [user introduction](./users/intro) page to learn what UnifiedPush is and get instructions to use it.
-
-### Which apps does UnifiedPush work with?
+## TL;DR
 
-Apps need to support the UnifiedPush protocol before they can use self-hosted push notifications. See [the application list](users/apps). If an app isn't in there, feel free to ask in the UnifiedPush Matrix (bridged on Libera Chat IRC) or ask the application's dev team.
+* __I JUST WANT TO KNOW WHAT TO INSTALL__: [__Ntfy (Android)__](/users/distributors/ntfy/)
+* The same but I must use Google for some reason (hard battery optimization): [UP-FCM Distrib (Android)](/users/distributors/fcm/)
+* I have a nextcloud server: [NextPush (Android)](/users/distributors/nextpush/)
+* Else, check behind.
```
Comment (binwiederhier) [inline content/_index.md:11]: For other options, see below

File: content/_index.md [hunk 2]
```diff
@@ -3,37 +3,25 @@ title: UnifiedPush
...
-
-You should use the UnifiedPush [connector library](/developers/) for your framework. If it doesn't already exist for your framework/language, contributions are welcome.
-
-### Will UnifiedPush ever work on iOS?
-
-iOS doesn't support running services in the background, so running a UnifiedPush distributor won't be possible without jailbreaking or Apple's approval for the foreseeable future. If someone has a solution, share that in the chat.
+## Getting Started
 
-### Can we use transport X to deliver notifications to the device?
+### Uers
```
Comment (binwiederhier) [inline content/_index.md:15]: Users

File: content/_index.md [hunk 3]
```diff
@@ -3,37 +3,25 @@ title: UnifiedPush
...
+* [Introduction](/users/intro)
+* [FAQ](/users/faq)
+* [Choose a distributor](/users/distributors)
 
-### What is the difference between UnifiedPush and OpenPush?
+### Developers
 
-UnifiedPush was born in the OpenPush chat. There are some similar core ideas as well; but operationally, the projects are not related. The goals of the projects are the same, except UnifiedPush allows for various different distributors rather than one.
+* [Definitions](/spec/definitions)
+* [Use Unifiedpush for your app](/developers/intro)
```
Comment (binwiederhier) [inline content/_index.md:24]: UnifiedPush

File: content/users/distributors/ntfy.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+geekdocHidden: true
+geekdocHiddenTocTree: false
+title: Ntfy - Android
```
Comment (binwiederhier) [inline content/users/distributors/ntfy.md:4]: Semantics: "ntfy" is spelled all lower case, but the Google Play store yelled at me for that so the Android app is spelled "Ntfy" (which I hate). I think it's fine either way though.

I usually write "ntfy Android app" and "ntfy server"

File: content/users/distributors/ntfy.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+weight: 10
+---
+
+Ntfy is a push provider easy to set up that does not required an account.
```
Comment (binwiederhier) [inline content/users/distributors/ntfy.md:8]: "does not require an account"

File: content/users/distributors/ntfy.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+* License: Apache2 | GPLv2
+* Sources: <https://github.com/binwiederhier/ntfy>
+* Server: <https://github.com/UP-NextPush/server-app>
```
Comment (binwiederhier) [inline content/users/distributors/ntfy.md:12]:
```
* Sources: <https://github.com/binwiederhier/ntfy-android>
* Server: <https://github.com/binwiederhier/ntfy-android> (not required unless you want to self-host)
* Website <https://ntfy.sh>
```

File: content/users/distributors/ntfy.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+* website: <https://ntfy.sh/>
+* Technology: HTTP Server Sent Events
```
Comment (binwiederhier) [inline content/users/distributors/ntfy.md:14]: Technically, the Android app does a JSON stream, but that's semantics again with a slightly different format

File: content/users/distributors/ntfy.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+[<img alt="Get it on F-Droid" src="/img/f-droid-badge.png" height=100 >](https://f-droid.org/packages/io.heckel.ntfy)
+
+## Set Up
+
+* Install this app on your device.
+* You're ready!
+
+## Self Hosting
+
+If you want to self host the server, you can find the steps here: https://ntfy.sh/docs/install/
```
Comment (binwiederhier) [inline content/users/distributors/ntfy.md:26]: Make this a link? Or does this automatically turn into a link?

File: content/users/intro.md [hunk 9]
```diff
@@ -3,50 +3,51 @@ title: Introduction
...
-### Easy - Use UnifiedPush hosted server
+### Without self-hosting the server (Easy)
 
-1. Install [the Gotify-UP app](../distributors/gotify)   
-  * Note: `Gotify-UP` is different from `Gotify` as the latter doesn't support UnifiedPush.
-2. Visit <https://gotify1.unifiedpush.org> and click on `Register`. Sign up with a username and secure password.
-3. Open the Gotify-UP app on your device, and login with those credentials.
-4. Now, you should be able to open any [apps supporting UnifiedPush](../apps), and should be able to receive push notifications!
-  * Note: if an app asks you which push distributor you want, you can now pick Gotify (though it probably shouldn't ask you).
+* [Ntfy](/users/distributors/ntfy): You just need to install it.
```
Comment (binwiederhier) [inline content/users/intro.md:22]: ntfy, not Ntfy (maybe)

File: content/users/intro.md [hunk 10]
```diff
@@ -3,50 +3,51 @@ title: Introduction
...
+* [UP-FCM Distributor](/user/distributors/fcm): Idem. Just be aware that it uses Google servers.
+* [Gotify](/users/distributors/gotify): You can use an opened to registration server.
 
-### Difficult - Self Host the server
+### Self-host the server (Difficult)
 
 Self hosting lets you have control of the server through which you receive push notifications. You have several distributor/provider options for self-hosting:
 
-#### [NextPush](../distributors/nextpush)
+#### [Ntfy](/users/distributors/nextpush)
```
Comment (binwiederhier) [inline content/users/intro.md:30]: `[ntfy](/users/distributors/ntfy)` (lower case, and link wrong!)

File: content/users/intro.md [hunk 11]
```diff
@@ -3,50 +3,51 @@ title: Introduction
...
+
+Ntfy server can be easily be self-hosted. It can be installed with a package, available for most linux distributions, with docker, or using the standalone binary.
```
Comment (binwiederhier) [inline content/users/intro.md:32]: "The ntfy server" (spelling and "the")

**Review by binwiederhier — APPROVED** [review]

File: content/users/distributors/ntfy.md (hunk 4 above)
Comment (p1gp1g) [reply content/users/distributors/ntfy.md:4]: It is as you want, I have started with an upper char for all distributor pages

**Comment by karmanyaahm** [comment]
Can you check out these changes? https://github.com/karmanyaahm/documentation/commit/068ba25d6f3cd8d517ce493fda7f71e693705cee
Also, I feel like the `What are push notifications?` FAQ should probably be kept on the home page because it'll hugely simplify the first experience people who don't know anything about push technology will have when looking at UnifiedPush. They won't have to go around on multiple pages just to understand what UnifiedPush is.

**Comment by p1gp1g** [comment]
> Can you check out these changes? [karmanyaahm@a98c61a](https://github.com/karmanyaahm/documentation/commit/a98c61a34f3c8b48ee9558d4bad537aee71c5eb3) Also, I feel like the `What are push notifications?` FAQ should probably be kept on the home page because it'll hugely simplify the first experience people who don't know anything about push technology will have when looking at UnifiedPush. They won't have to go around on multiple pages just to understand what UnifiedPush is.

I've pushed but we can add an asterisk for the push notification on the index

**Pull request merged**